README.md

```diff
@@ -37,15 +37,33 @@ JAVA&nbsp;7 when working with this project.
      the following two commands in separate shells:
 
        ```
-       $ sh run_server.sh
-       $ sh run_client.sh
+       $ sh run_server.sh <team_id> <team_secret> <port>
+       $ sh run_client.sh <host> <port>
        ```
 
-     The `run_server` and `run_client` scripts have hard-coded addresses for
-     your local machine. If you are running the server on a different machine
-     than the client, you will need to change the host portion of the address
-     in `run_client.sh` to the name of the host where your server is running.
-     Make sure the client and server are using the same port number.
+     You must specify the following startup arguments for `run_server.sh:
+     + `<team_id>` and `<team_secret>`: a numeric id for your team, and a secret
+       code, which are used to authenticate your server with the Relay server.
```

Comment:
The server takes any  string as input, and unless one connects to the Relay the secret is never used. Plus Secret.parse() can handle non-hex characters, so any char out of range will be seen as 0, but I'll add the specs anyway :)